Client/Frontend/Browser/URLBarView.swift

```diff
@@ -90,6 +91,47 @@ class URLBarView: UIView {
         }
     }
 
+    // The transition between the URL bar being fully displayed (1.0) and being minimised (0.0)
+    var state: CGFloat = 1.0 {
```

Comment:
Nit: State seems to generic of a word for this property. Maybe something more precise like `transitionValue`?